rel/config/config.exs

```diff
@@ -64,6 +64,7 @@ port = Utils.Environment.get("PORT")
 
 # General application configuration
 config :elixir_boilerplate,
+  version: Utils.Version.get(),
```

Comment:
Should we use this in the Healthcheck `plug` instead of `Application.spec(:elixir_boilerplate, :vsn) |> tostring()`?

Reply:
Yep, good catch! I’ll change it in this branch.